Data/Sys/GameSettings/FAA.ini

```diff
@@ -1,26 +1,4 @@
 # FAAE01, FAAP01 - Donkey Kong
 
-[Core]
-# Values set here will override the main Dolphin settings.
-
-[EmuState]
-# The Emulation State. 1 is worst, 5 is best, 0 is not set.
-EmulationStateId = 4
-EmulationIssues = Texture filtering will cause glitches.
-
-[OnLoad]
-# Add memory patches to be loaded once on boot here.
-
-[OnFrame]
-# Add memory patches to be applied every frame here.
-
-[ActionReplay]
-# Add action replay cheats here.
-
-[Video_Settings]
-SafeTextureCacheColorSamples = 512
-EFBScale = 2
-
-[Video_Enhancements]
-MaxAnisotropy = 0
-ForceFiltering = False
+[Video_Hacks]
```

Comment:
Ah ok, sorry my bad, the line was hidden at the online viewer and didn't notice it. You can disregard all these comments.